lib/RSAKeyPair.js

```diff
@@ -24,18 +53,45 @@ class RSAKeyPair extends LDKeyPair {
 
     this.validateKeyParams(); // validate keyBits and exponent
   }
-
+  /**
+   * @description Returns the public key.
+   * @example
+   * > rsaKeyPair.publicKey;
+   * 3423432423
+   * @implements {LDKeyPair#publicKey}
+   * @readonly
+   * @returns {string} - Returns the public key.
+   * @see [publicKey]{@link ./LDKeyPair.md#publicKey}
+   */
   get publicKey() {
     return this.publicKeyPem;
   }
-
+  /**
+   * @description Returns the private key.
+   * @example
+   * > rsaKeyPair.privateKey;
```

Comment:
this is a function, we really don't want to be getting into putting real RSA keys into examples.  Not sure an example is needed on a function call that takes no arguments (here and elsewhere).

Reply:
Removing the real RSA key sure although it was mentioned somewhere that examples should work. I think the example is fine we demonstrate that a get function can be called like an express.

Reply:
@davidlehn I believe you wanted real keys in examples so can you weigh in here? I went through the Ed2555189KeyPair examples last night and they actually work so I would like the RSA examples to work as well. let me know.